### PR TITLE
Health insurance move application under templates

### PR DESCRIPTION
**Pull request by qdd1**
# \<Description\>
Need to take out 2 console.log when merged bucket codes

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/health-insurance/src/lib/soap/healthInsuranceAPI.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        }
+      } else if (typeof value == 'object') {
+        xml += this.OBJtoXML(new Object(value))
+      } else {
+        xml += value
+      }
+      xml += value instanceof Array ? '' : '</' + key + '>'
+    })
+    return xml
+  }
```
Comment (eirikurn) [inline libs/health-insurance/src/lib/soap/healthInsuranceAPI.ts:221]: This is not a safe xml serializer, eg there's no escaping done.

Is the "skjal" a standalone XML document, i.e. not part of the WSDL?

File: libs/api/domains/health-insurance/src/lib/soap/healthInsuranceAPI.ts [hunk 2]
```diff
@@ -131,7 +131,7 @@ export class HealthInsuranceAPI {
     const vistaSkjalBody: GetVistaSkjalBody = {
       sjukratryggingumsokn: {
         einstaklingur: {
-          kennitala: nationalId,
+          kennitala: '0101671089', //nationalId,
```
Comment (eirikurn) [inline libs/api/domains/health-insurance/src/lib/soap/healthInsuranceAPI.ts:134]: Remove?

**Review by eirikurn — APPROVED** [review]

**Review by essgje — APPROVED** [review]
lftm

File: libs/health-insurance/src/lib/soap/healthInsuranceAPI.ts (hunk 1 above)
Comment (qdd1) [reply libs/health-insurance/src/lib/soap/healthInsuranceAPI.ts:221]: yes, its is. It's a standalone XML

File: libs/api/domains/health-insurance/src/lib/soap/healthInsuranceAPI.ts (hunk 2 above)
Comment (qdd1) [reply libs/api/domains/health-insurance/src/lib/soap/healthInsuranceAPI.ts:134]: Will remove it on next PR

**Pull request merged**